apps/fz_http/lib/fz_http_web/templates/shared/devices_table.html.heex

```diff
@@ -18,14 +18,14 @@
     <%= for device <- @devices do %>
       <tr>
         <td>
-          <.link navigate={Routes.device_admin_show_path(@socket, :show, device)}>
+          <.link navigate={~p"/devices/#{device}"}>
             <%= device.name %>
           </.link>
         </td>
         <%= if @show_user do %>
           <td>
             <%= live_redirect(device.user.email,
-              to: Routes.user_show_path(@socket, :show, device.user)
+to: ~p"/users/#{device.user}"
```

Comment:
Did the tabbing get squashed here?

Reply:
Whoops! Looks like the formatter isn’t catching these for some reason

Reply:
Yep, the formatter wasn't receiving the correct file list. Fixed.